workspaces/homepage/plugins/dynamic-home-page/src/components/QuickAccessCard.tsx

```diff
@@ -89,13 +86,7 @@ export const QuickAccessCard = (props: QuickAccessCardProps) => {
             title={item.title}
             tools={item.links.map(link => ({
               ...link,
-              icon: (
-                <img
-                  className={classes.img}
-                  src={link.iconUrl}
-                  alt={link.label}
-                />
-              ),
+              icon: <QuickAccessIcon icon={link.iconUrl} alt={link.label} />,
```

Comment:
Nit - Keeping the prop name `iconUrl` doesn't fully explains that it now supports more than just URLs for the icon. But I guess this will require bigger changes in documentation and RHDH defaults as well.

Reply:
@rohitkrai03 Yes, this definitely requires a documentation update in the RHDH docs. The `dynamic-home-page` plugin under `rhdh-plugins` has no readme.

Reply:
FYI @jmagak ^^